docs/source/setup/manual_way.rst

```diff
@@ -59,7 +64,52 @@ You are almost there, but there is no worker running yet.
 
 11. Now you should be able to select text in OCRed document!
 
-
 .. figure:: ../img/select_text.png
 
-   Now you should be able to select text
+   Now you should be able to select text.
+
+## Start Papermerge at boot
```

Comment:
this part is too specific. Start at boot of what ? These instructions would be more appropriate for specific distribution deployment (as part of that).

Reply:
I copied the idea from here  https://paperless.readthedocs.io/en/latest/setup.html#making-things-a-little-more-permanent  if it is already installed and running why not using it?
And as this is setup guide, why not guiding it?